### PR TITLE
first go at readonly

### PR DESCRIPTION
**Pull request by christophe-g**
Adding a `readonly` behavior for `paper-dropdown-menu`

- `paper-menu` is set to `disabled` mode
- the menu closes when setting to `readonly`

### CODE REVIEW

File: paper-dropdown-menu.html [hunk 1]
```diff
@@ -242,6 +242,16 @@
...
+          },
+
+          /**
+           * If you're using PaperInputBehavior to implement your own paper-input-like
+           * element, bind this to the `<input is="iron-input">`'s `readonly` property.
+           */
+          readonly: {
+            type: Boolean,
+            value: false,
+            observer: '_readonlyChanged'
```
Comment (valdrinkoshi) [inline paper-dropdown-menu.html]: I believe we can get rid of this observer, as it can be expensive (this is triggered for all the dropdown menus) and it actually solves a very edge case scenario where someone updates this property while the dropdown is opened.
WDYT?

**Comment by valdrinkoshi** [comment]
Hi @christophe-g, the implementation LGTM. 
Tests are failing because you need to sync your branch with `master`.
Could you also add some unit tests so we avoid future regressions?

File: paper-dropdown-menu.html (hunk 1 above)
Comment (christophe-g) [inline paper-dropdown-menu.html]: Thanks for the comment. 
Agreed, this is not going to happen very often (`readonly` property changing while the dropdown is opened), but it should still be the correct behavior, I guess. 
What is the actual weight of adding an observer ?

Comment (valdrinkoshi) [inline paper-dropdown-menu.html]: For the actual weight, `_readonlyChanged` callback is called the first time when `readonly` is set to its default value, and each time it changes value, which means when is set to true, we have `_readonlyChanged` called twice.
If you remove the default value for the `readonly` property, `_readonlyChanged` will be called only once, when we it is set the first time.

Comment (christophe-g) [inline paper-dropdown-menu.html]: Sounds like a good idea ! 
98b0a21

**Comment by valdrinkoshi** [comment]
Thanks for the contribution @christophe-g! can you add some unit tests to avoid any future regression?
